composer.json

```diff
@@ -3,7 +3,7 @@
     "minimum-stability": "stable",
     "require": {
         "php": ">=7.4",
-        "smarty/smarty": "^v4.3.1",
+        "smarty/smarty": "^v5.4.0",
```

Comment:
It looks like we're actually explicitly targeting v5.5.0 based on the discussion in the PR; should the title be adjusted and this changed to 5.5.0?

Reply:
The PR title isn't particularly important; the `composer.json` file probably also isn't because any install will use the locked version v5.5.0 from `composer.lock`. 

This version constraint permits any 5.* version >= 5.4.0, so it could be argued that since we depend on 5.5.0 specifically for the reverted deprecations we should specify that version. Realistically though, our locked version is always going to be above the specified minimum version in composer.json simply because of Dependabot. I'm minded to change this anyway, but it feels like a nitpick too. What do you think?